js/index.js

```diff
@@ -1,6 +1,36 @@
 //GLOBAL VARIABLES
 let messageId = 1;
 let activeLinkCounter = 0;
+//XMLHttpRequest Class
+class apiRequest {
```

Comment:
Since the only thing this class does for is call `open` on the request object for us, 
I'd either give it more responsibility or ditch it entirely. 
If you want to give it more responsibility to make it more useful to have around, maybe have it also handle attaching event listeners.

Reply:
I will make sure to give it more responsibility on the next lesson with fetch() API.